## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 448691`
Commit: [`097e92f976470390972e60de6cebca0d25d595d8`](https://github.com/NixOS/nixpkgs/commit/097e92f976470390972e60de6cebca0d25d595d8) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/097e92f976470390972e60de6cebca0d25d595d8..pull/448691/head))
Merge: [`ebb9daae505fd65afbc5d2933d421d1e80344403`](https://github.com/NixOS/nixpkgs/commit/ebb9daae505fd65afbc5d2933d421d1e80344403)

Logs: https://github.com/mdaniels5757/nixpkgs-review-gha/actions/runs/18251324385


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>zabbix70.agent</li>
    <li>zabbix70.agent2</li>
    <li>zabbix70.proxy-mysql</li>
    <li>zabbix70.proxy-pgsql</li>
    <li>zabbix70.proxy-sqlite</li>
    <li>zabbix70.server (zabbix70.server-pgsql)</li>
    <li>zabbix70.server-mysql</li>
    <li>zabbix70.web</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>zabbix70.agent</li>
    <li>zabbix70.agent2</li>
    <li>zabbix70.proxy-mysql</li>
    <li>zabbix70.proxy-pgsql</li>
    <li>zabbix70.proxy-sqlite</li>
    <li>zabbix70.server (zabbix70.server-pgsql)</li>
    <li>zabbix70.server-mysql</li>
    <li>zabbix70.web</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>zabbix70.agent2</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>zabbix70.agent</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>zabbix70.agent2</summary>
<pre>clang -DHAVE_CONFIG_H -I../../include/common -I../../include    -DZABBIX_DAEMON -DDEFAULT_CONFIG_FILE=&quot;\&quot;/nix/store/70k7dbzw83rfkndq3d6j0v9aisrh36sh-zabbix-agent2-7.0.18/etc/zabbix_agentd.conf\&quot;&quot; -DDEFAULT_LOAD_MODULE_PATH=&quot;\&quot;/nix/store/70k7dbzw83rfkndq3d6j0v9aisrh36sh-zabbix-agent2-7.0.18/lib/modules\&quot;&quot; -I/nix/store/axjwrz52dkhrvkgvyy8kfprkhdxybjdr-openssl-3.4.2-dev/include -g -O2 -I/nix/store/k5a0m5zi9lracknxx5i7fjdcsxay0m19-pcre-8.45-dev/include -MT zabbix_agentd-zabbix_agentd.o -MD -MP -MF .deps/zabbix_agentd-zabbix_agentd.Tpo -c -o zabbix_agentd-zabbix_agentd.o `test -f &#x27;zabbix_agentd.c&#x27; || echo &#x27;./&#x27;`zabbix_agentd.c
mv -f .deps/zabbix_agentd-zabbix_agentd.Tpo .deps/zabbix_agentd-zabbix_agentd.Po
clang -DZABBIX_DAEMON -DDEFAULT_CONFIG_FILE=&quot;\&quot;/nix/store/70k7dbzw83rfkndq3d6j0v9aisrh36sh-zabbix-agent2-7.0.18/etc/zabbix_agentd.conf\&quot;&quot; -DDEFAULT_LOAD_MODULE_PATH=&quot;\&quot;/nix/store/70k7dbzw83rfkndq3d6j0v9aisrh36sh-zabbix-agent2-7.0.18/lib/modules\&quot;&quot; -I/nix/store/axjwrz52dkhrvkgvyy8kfprkhdxybjdr-openssl-3.4.2-dev/include -g -O2 -I/nix/store/k5a0m5zi9lracknxx5i7fjdcsxay0m19-pcre-8.45-dev/include -L/nix/store/axjwrz52dkhrvkgvyy8kfprkhdxybjdr-openssl-3.4.2-dev/lib   -rdynamic  -L/nix/store/g9hsqyr2hrnvryvj8w0hn8hrr0chfw28-pcre-8.45/lib -L/usr/lib -o zabbix_agentd zabbix_agentd-zabbix_agentd.o active_checks/libzbxactive_checks.a listener/libzbxlistener.a agent_conf/libagent_conf.a logfiles/libzbxlogfiles.a metrics/libzbxmetrics.a ../../src/libs/zbxsysinfo/libzbxagentsysinfo.a ../../src/libs/zbxsysinfo/osx/libfunclistsysinfo.a ../../src/libs/zbxsysinfo/osx/libspechostnamesysinfo.a ../../src/libs/zbxsysinfo/agent/libagentsysinfo.a ../../src/libs/zbxsysinfo/common/libcommonsysinfo.a ../../src/libs/zbxsysinfo/common/libcommonsysinfo_httpmetrics.a ../../src/libs/zbxsysinfo/osx/libspecsysinfo.a ../../src/libs/zbxsysinfo/common/libcommonsysinfo_http.a ../../src/libs/zbxsysinfo/simple/libsimplesysinfo.a ../../src/libs/zbxsysinfo/alias/libalias.a ../../src/libs/zbxlog/libzbxlog.a ../../src/libs/zbxregexp/libzbxregexp.a ../../src/libs/zbxthreads/libzbxthreads.a ../../src/libs/zbxmutexs/libzbxmutexs.a ../../src/libs/zbxprof/libzbxprof.a ../../src/libs/zbxnix/libzbxnix.a ../../src/libs/zbxcomms/libzbxcomms.a ../../src/libs/zbxcommshigh/libzbxcommshigh.a ../../src/libs/zbxcfg/libzbxcfg.a ../../src/libs/zbxjson/libzbxjson.a ../../src/libs/zbxalgo/libzbxalgo.a ../../src/libs/zbxvariant/libzbxvariant.a ../../src/libs/zbxcommon/libzbxcommon.a ../../src/libs/zbxgetopt/libzbxgetopt.a ../../src/libs/zbxhash/libzbxhash.a ../../src/libs/zbxcrypto/libzbxcrypto.a ../../src/libs/zbxexec/libzbxexec.a ../../src/libs/zbxcompress/libzbxcompress.a ../../src/libs/zbxmodules/libzbxmodules.a ../../src/libs/zbxhttp/libzbxhttp.a ../../src/libs/zbxfile/libzbxfile.a ../../src/libs/zbxip/libzbxip.a ../../src/libs/zbxtime/libzbxtime.a ../../src/libs/zbxnum/libzbxnum.a ../../src/libs/zbxstr/libzbxstr.a ../../src/libs/zbxparam/libzbxparam.a ../../src/libs/zbxexpr/libzbxexpr.a ../../src/libs/zbxcurl/libzbxcurl.a ../../src/libs/zbxbincommon/libzbxbincommon.a -lssl -lcrypto   -lpcre -lpthread -lm  -lresolv -liconv
make[3]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/zabbix_agent&#x27;
make[2]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/zabbix_agent&#x27;
Making all in go
make[2]: Entering directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/go&#x27;
CGO_CFLAGS=&quot;-I/nix/store/axjwrz52dkhrvkgvyy8kfprkhdxybjdr-openssl-3.4.2-dev/include -I/nix/store/k5a0m5zi9lracknxx5i7fjdcsxay0m19-pcre-8.45-dev/include &quot; CGO_LDFLAGS=&quot;-L/nix/store/axjwrz52dkhrvkgvyy8kfprkhdxybjdr-openssl-3.4.2-dev/lib  		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxalgo/libzbxalgo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxinterface/libzbxinterface.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxcommon/libzbxcommon.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxcomms/libzbxcomms.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxcompress/libzbxcompress.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxcrypto/libzbxcrypto.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxexec/libzbxexec.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxhash/libzbxhash.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxhttp/libzbxhttp.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxjson/libzbxjson.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxmutexs/libzbxmutexs.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxprof/libzbxprof.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxnix/libzbxnix.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxregexp/libzbxregexp.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxnum/libzbxnum.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxstr/libzbxstr.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxfile/libzbxfile.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxparam/libzbxparam.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxexpr/libzbxexpr.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxtime/libzbxtime.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxip/libzbxip.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxsysinfo/osx/libspechostnamesysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxsysinfo/osx/libspecsysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxsysinfo/alias/libalias.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxsysinfo/common/libcommonsysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxsysinfo/libzbxagent2sysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxsysinfo/simple/libsimplesysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxthreads/libzbxthreads.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxvariant/libzbxvariant.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/libs/zbxxml/libzbxxml.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/zabbix_agent/logfiles/libzbxlogfiles.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/zabbix_agent/active_checks/libzbxactive_checks.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/zabbix_agent/metrics/libzbxmetrics.a 		-lpcre -lpthread -lm  -lresolv -liconv    -lssl -lcrypto &quot; \
	go build \
	-tags=&quot;pcre&quot; \
	-ldflags=&quot;-X &#x27;golang.zabbix.com/agent2/pkg/version.compileDate=Jan 1 1970&#x27; -X golang.zabbix.com/agent2/pkg/version.compileTime=00:00:00 -X golang.zabbix.com/agent2/pkg/version.compileOs=darwin -X golang.zabbix.com/agent2/pkg/version.compileArch=amd64 -X main.confDefault=/nix/store/70k7dbzw83rfkndq3d6j0v9aisrh36sh-zabbix-agent2-7.0.18/etc/zabbix_agent2.conf -X main.applicationName=zabbix_agent2&quot; \
	-o bin \
	golang.zabbix.com/agent2/cmd/zabbix_agent2
# golang.zabbix.com/agent2/plugins/smart
plugins/smart/smartfs.go:247:43: p.getOsDevices undefined (type *Plugin has no field or method getOsDevices)
make[2]: *** [Makefile:629: bin/zabbix_agent2] Error 1
make[2]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src/go&#x27;
make[1]: *** [Makefile:507: all-recursive] Error 1
make[1]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-28267-529683681/zabbix-7.0.18/src&#x27;
make: *** [Makefile:559: all-recursive] Error 1</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>zabbix70.agent2</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>zabbix70.agent</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>zabbix70.agent2</summary>
<pre>clang -DHAVE_CONFIG_H -I../../include/common -I../../include    -DZABBIX_DAEMON -DDEFAULT_CONFIG_FILE=&quot;\&quot;/nix/store/h50hbmizhbrbzslqi9s3pwflqxnya90n-zabbix-agent2-7.0.18/etc/zabbix_agentd.conf\&quot;&quot; -DDEFAULT_LOAD_MODULE_PATH=&quot;\&quot;/nix/store/h50hbmizhbrbzslqi9s3pwflqxnya90n-zabbix-agent2-7.0.18/lib/modules\&quot;&quot; -I/nix/store/jr12cbbmviv009pl9qq24jak5dbiw2bc-openssl-3.4.2-dev/include -g -O2 -I/nix/store/9y0m3sprv7z6qk67xfs4llpqlir6ilx7-pcre-8.45-dev/include -MT zabbix_agentd-zabbix_agentd.o -MD -MP -MF .deps/zabbix_agentd-zabbix_agentd.Tpo -c -o zabbix_agentd-zabbix_agentd.o `test -f &#x27;zabbix_agentd.c&#x27; || echo &#x27;./&#x27;`zabbix_agentd.c
mv -f .deps/zabbix_agentd-zabbix_agentd.Tpo .deps/zabbix_agentd-zabbix_agentd.Po
clang -DZABBIX_DAEMON -DDEFAULT_CONFIG_FILE=&quot;\&quot;/nix/store/h50hbmizhbrbzslqi9s3pwflqxnya90n-zabbix-agent2-7.0.18/etc/zabbix_agentd.conf\&quot;&quot; -DDEFAULT_LOAD_MODULE_PATH=&quot;\&quot;/nix/store/h50hbmizhbrbzslqi9s3pwflqxnya90n-zabbix-agent2-7.0.18/lib/modules\&quot;&quot; -I/nix/store/jr12cbbmviv009pl9qq24jak5dbiw2bc-openssl-3.4.2-dev/include -g -O2 -I/nix/store/9y0m3sprv7z6qk67xfs4llpqlir6ilx7-pcre-8.45-dev/include -L/nix/store/jr12cbbmviv009pl9qq24jak5dbiw2bc-openssl-3.4.2-dev/lib   -rdynamic  -L/nix/store/zdbrsgqqh227f3iajfayzwmvpqd4yqkx-pcre-8.45/lib -L/usr/lib -o zabbix_agentd zabbix_agentd-zabbix_agentd.o active_checks/libzbxactive_checks.a listener/libzbxlistener.a agent_conf/libagent_conf.a logfiles/libzbxlogfiles.a metrics/libzbxmetrics.a ../../src/libs/zbxsysinfo/libzbxagentsysinfo.a ../../src/libs/zbxsysinfo/osx/libfunclistsysinfo.a ../../src/libs/zbxsysinfo/osx/libspechostnamesysinfo.a ../../src/libs/zbxsysinfo/agent/libagentsysinfo.a ../../src/libs/zbxsysinfo/common/libcommonsysinfo.a ../../src/libs/zbxsysinfo/common/libcommonsysinfo_httpmetrics.a ../../src/libs/zbxsysinfo/osx/libspecsysinfo.a ../../src/libs/zbxsysinfo/common/libcommonsysinfo_http.a ../../src/libs/zbxsysinfo/simple/libsimplesysinfo.a ../../src/libs/zbxsysinfo/alias/libalias.a ../../src/libs/zbxlog/libzbxlog.a ../../src/libs/zbxregexp/libzbxregexp.a ../../src/libs/zbxthreads/libzbxthreads.a ../../src/libs/zbxmutexs/libzbxmutexs.a ../../src/libs/zbxprof/libzbxprof.a ../../src/libs/zbxnix/libzbxnix.a ../../src/libs/zbxcomms/libzbxcomms.a ../../src/libs/zbxcommshigh/libzbxcommshigh.a ../../src/libs/zbxcfg/libzbxcfg.a ../../src/libs/zbxjson/libzbxjson.a ../../src/libs/zbxalgo/libzbxalgo.a ../../src/libs/zbxvariant/libzbxvariant.a ../../src/libs/zbxcommon/libzbxcommon.a ../../src/libs/zbxgetopt/libzbxgetopt.a ../../src/libs/zbxhash/libzbxhash.a ../../src/libs/zbxcrypto/libzbxcrypto.a ../../src/libs/zbxexec/libzbxexec.a ../../src/libs/zbxcompress/libzbxcompress.a ../../src/libs/zbxmodules/libzbxmodules.a ../../src/libs/zbxhttp/libzbxhttp.a ../../src/libs/zbxfile/libzbxfile.a ../../src/libs/zbxip/libzbxip.a ../../src/libs/zbxtime/libzbxtime.a ../../src/libs/zbxnum/libzbxnum.a ../../src/libs/zbxstr/libzbxstr.a ../../src/libs/zbxparam/libzbxparam.a ../../src/libs/zbxexpr/libzbxexpr.a ../../src/libs/zbxcurl/libzbxcurl.a ../../src/libs/zbxbincommon/libzbxbincommon.a -lssl -lcrypto   -lpcre -lpthread -lm  -lresolv -liconv
make[3]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/zabbix_agent&#x27;
make[2]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/zabbix_agent&#x27;
Making all in go
make[2]: Entering directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/go&#x27;
CGO_CFLAGS=&quot;-I/nix/store/jr12cbbmviv009pl9qq24jak5dbiw2bc-openssl-3.4.2-dev/include -I/nix/store/9y0m3sprv7z6qk67xfs4llpqlir6ilx7-pcre-8.45-dev/include &quot; CGO_LDFLAGS=&quot;-L/nix/store/jr12cbbmviv009pl9qq24jak5dbiw2bc-openssl-3.4.2-dev/lib  		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxalgo/libzbxalgo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxinterface/libzbxinterface.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxcommon/libzbxcommon.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxcomms/libzbxcomms.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxcompress/libzbxcompress.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxcrypto/libzbxcrypto.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxexec/libzbxexec.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxhash/libzbxhash.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxhttp/libzbxhttp.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxjson/libzbxjson.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxmutexs/libzbxmutexs.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxprof/libzbxprof.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxnix/libzbxnix.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxregexp/libzbxregexp.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxnum/libzbxnum.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxstr/libzbxstr.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxfile/libzbxfile.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxparam/libzbxparam.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxexpr/libzbxexpr.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxtime/libzbxtime.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxip/libzbxip.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxsysinfo/osx/libspechostnamesysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxsysinfo/osx/libspecsysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxsysinfo/alias/libalias.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxsysinfo/common/libcommonsysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxsysinfo/libzbxagent2sysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxsysinfo/simple/libsimplesysinfo.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxthreads/libzbxthreads.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxvariant/libzbxvariant.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/libs/zbxxml/libzbxxml.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/zabbix_agent/logfiles/libzbxlogfiles.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/zabbix_agent/active_checks/libzbxactive_checks.a 		/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/zabbix_agent/metrics/libzbxmetrics.a 		-lpcre -lpthread -lm  -lresolv -liconv    -lssl -lcrypto &quot; \
	go build \
	-tags=&quot;pcre&quot; \
	-ldflags=&quot;-X &#x27;golang.zabbix.com/agent2/pkg/version.compileDate=Jan 1 1970&#x27; -X golang.zabbix.com/agent2/pkg/version.compileTime=00:00:00 -X golang.zabbix.com/agent2/pkg/version.compileOs=darwin -X golang.zabbix.com/agent2/pkg/version.compileArch=arm64 -X main.confDefault=/nix/store/h50hbmizhbrbzslqi9s3pwflqxnya90n-zabbix-agent2-7.0.18/etc/zabbix_agent2.conf -X main.applicationName=zabbix_agent2&quot; \
	-o bin \
	golang.zabbix.com/agent2/cmd/zabbix_agent2
# golang.zabbix.com/agent2/plugins/smart
plugins/smart/smartfs.go:247:43: p.getOsDevices undefined (type *Plugin has no field or method getOsDevices)
make[2]: *** [Makefile:629: bin/zabbix_agent2] Error 1
make[2]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src/go&#x27;
make[1]: *** [Makefile:507: all-recursive] Error 1
make[1]: Leaving directory &#x27;/nix/build/nix-build-zabbix-agent2-7.0.18.drv-74477-3374573466/zabbix-7.0.18/src&#x27;
make: *** [Makefile:559: all-recursive] Error 1</pre>
</details>
</details>

--- a/pkgs/servers/monitoring/zabbix/versions.nix
+++ b/pkgs/servers/monitoring/zabbix/versions.nix
@@ -4,8 +4,8 @@ generic: {
     hash = "sha256-US+TP6qtCT3LlnELWR93t7nf8A1Y1xRPI+300GA1v8g=";
   };
   v70 = generic {
-    version = "7.0.18";
-    hash = "sha256-kSXO4hRB/9qqGJPuEjJz740DChUJCGz63ozdCxgCupg=";
+    version = "7.0.19";
+    hash = "sha256-ML7wFzSTsZk3fJBhs06KLhaijrDW9+nHuUJDPkt1Nn8=";
   };
   v60 = generic {
     version = "6.0.36";

